<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="6" hostname="HC-5ZBQ6S3" failures="2" tests="2" name="Regression" time="0.04" errors="0" timestamp="2024-05-20T15:51:15 IST">
  <testcase classname="test.test1" name="secondDemo" time="0.007">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.test1.secondDemo(test1.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- secondDemo -->
  <testcase classname="test.test2" name="test2SecondDemo" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.test2.test2SecondDemo(test2.java:18)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test2SecondDemo -->
  <testcase classname="test.test4" name="test4FirstDemo" time="0.0">
    <ignored/>
  </testcase> <!-- test4FirstDemo -->
  <testcase classname="test.test4" name="test4SecondDemo" time="0.0">
    <ignored/>
  </testcase> <!-- test4SecondDemo -->
  <testcase classname="test.test2" name="test2FirstDemo" time="0.0">
    <ignored/>
  </testcase> <!-- test2FirstDemo -->
  <testcase classname="test.test3" name="test3SecondDemo" time="0.0">
    <ignored/>
  </testcase> <!-- test3SecondDemo -->
  <testcase classname="test.test1" name="zfirstDemo" time="0.0">
    <ignored/>
  </testcase> <!-- zfirstDemo -->
  <testcase classname="test.test3" name="test3FirstDemo" time="0.0">
    <ignored/>
  </testcase> <!-- test3FirstDemo -->
</testsuite> <!-- Regression -->
